src/pages/workspace/accounting/qbo/import/QuickbooksTaxesPage.tsx

```diff
@@ -44,10 +46,12 @@ function QuickbooksTaxesPage({policy}: WithPolicyProps) {
                                     accessibilityLabel={translate('workspace.accounting.taxes')}
                                     isOn={!!syncTax}
                                     onToggle={() => Connections.updatePolicyConnectionConfig(policyID, CONST.POLICY.CONNECTIONS.NAME.QBO, CONST.QUICK_BOOKS_CONFIG.SYNC_TAX, !syncTax)}
+                                    disabled={isJournalExportEntity}
                                 />
                             </View>
                         </OfflineWithFeedback>
                     </View>
+                    {isJournalExportEntity && <Text style={[styles.mutedNormalTextLabel, styles.pt2]}>{translate('workspace.qbo.taxesJournalEntrySwitchNote')}</Text>}
```

Comment:
The hint text does not seem right for the above-mentioned scenario i.e. when the `syncTax` is true 
 `Journal Entry` export is set). 
A separate hint text may be better here for this specific scenario
What do you think?

Reply:
> Something like !syncTax && isJournalExportEntity could solve this problem.

Yes I agree.

@teneeto, could you remind me if we have an error message on the frontend side for the rare case of Journal Entry is selected while the `syncTax` is one (when the configuration was changed in Expensify Classic, etc).

Reply:
> @teneeto, could you remind me if we have an error message on the frontend side for the rare case of Journal Entry is selected while the syncTax is one (when the configuration was changed in Expensify Classic, etc).

Checked the code, and we display [this error message](https://github.com/Expensify/App/blob/59b4f1a995beafed2704055f6b14c51de71ae5c0/src/languages/en.ts#L2021) when the above misconfiguration happens [here](https://github.com/Expensify/App/blob/0baa2147fbcbbd964394134c7c82fed7fda1f0b9/src/pages/workspace/accounting/qbo/export/QuickbooksOutOfPocketExpenseConfigurationPage.tsx#L42-L49)

Reply:
Or do we already have an error message for the above case, too?

Reply:
Let me confirm

Reply:
bump on this @teneeto 🙇

Reply:
!syncTax && isJournalExportEntity

Reply:
Got this from the initial [issue description](https://github.com/Expensify/App/issues/41042#issue-2264625299) and follow-up comment by @trjExpensify https://github.com/Expensify/App/issues/41042#issuecomment-2083614043

Reply:
>@teneeto, or do we already have an error message for the above case, too?
There are no error fields here. should we add here?

Reply:
> There are no error fields here. should we add here?

Please see the comment below 😄 

> we will probably move the error handling to the backend. We don't have to do this on the front end. Please ignore this comment ⬇️

Reply:
> Is there any change I'll need to make?

Thank you for reminding where the UI copy came from. I agree with @rojiphil on the following point:

> The hint text does not seem right for the above-mentioned scenario i.e. when the syncTax is true
Journal Entry export is set).

I think we should have the same condition as what @rojiphil suggested -> `!syncTax && isJournalExportEntity` to display this hint message

Reply:
ok

Reply:
@teneeto, can you modify the code according to my suggestion [here](https://github.com/Expensify/App/pull/41520/files#r1600314637)?

What the user needs to do in this case (`!syncTax && isJournalExportEntity`) is to toggle off the Switch and the hint text can be misleading

Reply:
it's rare case anyway, so I gonna address this change when I make a backend change to handle errors on the backend side 👍